backend/typescript/models/behaviour.model.ts

```diff
@@ -1,6 +1,6 @@
 import { Column, Model, Table } from "sequelize-typescript";
 
-@Table({ tableName: "behaviours" })
+@Table({ timestamps: false, tableName: "behaviours" })
```

Comment:
what does the timestamp do

Reply:
Sequelize automatically makes like a createdAt column and stuff so this line just stops it from doing that